src/pivotal-ui/components/forms/link-updater.js

```diff
@@ -20,14 +20,14 @@ LinkUpdater.prototype.updateValue = function(inputValue){
   if(err) {
     inputValue = this.currentValue;
     this.input.val(this.currentValue);
-    // setImmediate forces input-error to be triggered on the next tick, so it
+    // setTimeout 0 forces input-error to be triggered on the next tick, so it
     // will follow any input
-    setImmediate(function(){
+    setTimeout(function(){
       this.input.trigger({
         type: "input-error",
         message: err.message
       });
-    }.bind(this));
+    }.bind(this), 0);
```

Comment:
RAF _might_ be better here though. Just trying to reduce complexity whichever way possible.